protocol/wlr-screencopy-unstable-v1.xml

```diff
@@ -102,6 +102,12 @@
 
         The client should then create a buffer with the provided attributes, and
         send a "copy" request.
+
+        When the buffer is a dmabuf, the stride parameter may be ignored.
```

Comment:
It would be better to say that the compositor can send a zero stride to mean "you can choose whatever".

Reply:
How about this: "A stride parameter of 0 means that the stride is inconsequential to the operation of the compositor and may be chosen by the client."

Reply:
Oh, yeah, I forgot: We don't actually know that the buffer is going to be a dmabuf at this point, so the stride must contain a valid value.